File: Day19_algorithm/algorithm/algorithm1.js
let arr1 = [4,2,1,3,9,5,6];
let arr2 = [7,6,5,4,3,2,1];
let arr3 = [8,6,3,3,4,1,5,7];

function solution(arr,m){
  // 정렬
  arr.sort((a,b) => a-b);
  // 계산
  let sum = 0;
  for(let i =1; i<arr.length; i++){
    if(i%m === 0)
      sum += arr[i];
  }
  console.log(`${m}의 배수 인덱스 합: ${sum}`);
}

solution(arr1,3);
solution(arr2,3);
solution(arr3,2);

File: Day19_algorithm/algorithm/algorithm1.py
arr = list(map(int,input().split()))

# 예외처리
while True:
  m = int(input('배수가 될 m을 입력하세요: '))
  if m > 0 :
    break
  print('다시 입력하세요')

# 배열 정렬
arr.sort()
print(arr)

sum = 0
for i in range(1, len(arr)):
  if i % m == 0 :
    sum += arr[i]

print(sum)
  